projects/plugins/jetpack/to-test.md

```diff
@@ -1,30 +1,38 @@
-## 10.5
+## 10.6
 
 ### Before you start
 
 - **At any point during your testing, remember to [check your browser's JavaScript console](https://codex.wordpress.org/Using_Your_Browser_to_Diagnose_JavaScript_Errors#Step_3:_Diagnosis) and see if there are any errors reported by Jetpack there.**
 - Use "Debug Bar" or "Query Monitor" to help make PHP notices and warnings more noticeable and report anything you see.
 
-### Updated Print Media Styles
+### Instant Search
 
-When printing posts, additional interactive elements not meant for print have been hidden. To test:
+In this release, we've continued to make progress towards a Jetpack Search plugin. This involved making changes to the Search feature as it is packaged in the Jetpack plugin, so it's worth testing it still works as advertized. Try the following:
 
-- In `Jetpack > Settings > Traffic` enable the Related Posts feature, and in `Jetpack > Settings > Sharing` enable and configure sharing buttons. Also enable Likes for a particular post to test with.
-- When printing your test post, you should not see the Like button, "Share this" section, or the list of Related Posts.
+1. Purchase a Search plan.
+2. Go to Jetpack > Settings > Performance and toggle the feature on and off.
+3. Go to Jetpack > Search and toggle the feature on and off.
+4. Click the link there to go and customize the search page.
+5. Check that your changes still look good on the site's frontend.
+6. In Appearance > Widgets, add widgets to the Jetpack Search Sidebar area, and ensure those widgets appear when you view the Search results on your site's frontend.
 
-### Subscription Form Block Custom Success Message
+### Pay with PayPal Block
 
-The success message for the Subscription Form block can now be customized from the editor's block settings:
+In this release we've fixed some bugs with this block. It's a good opportunity to test it again. You'll need to purchase a paid bundle first, such as the Complete plan. Then, head to Posts > Add New and try adding a Pay with PayPal block. Let us know if you run into any issues with layout in the editor, currencies, or layout on the site's frontend.
 
-- Make sure subscriptions are enabled in `Jetpack > Settings > Discussion`.
-- In a new test post, add a Subscription Form block and customize the "Success Message Text" from the block settings panel.
-- Save your post and test you are able to see the customized success message displayed on the frontend after submitting the form.
+### Tiled Gallery Block
 
-### Single Sign On Error Filters
+We've made changes to the Tiled Gallery block in this release, to ensure it can be used from the mobile apps as well. You'll want to make sure it can still be used from the block editor on the web:
```

Comment:
Does the mobile tiled gallery need testing at this point, or just web based editor?